docs/photutils/psf.rst

```diff
@@ -28,10 +28,391 @@ model-fitting analysis, regardless to exactly what kind of model is actually
 being fit.  We take this road, using "PSF photometry" as shorthand for the
 general approach.
 
+PSF Photometry in Crowded Fields
+--------------------------------
+
+Photutils provides an implementation of the DAOPHOT algorithm
+(`~photutils.psf.DAOPhotPSFPhotometry`) proposed by Stetson in his
```

Comment:
could you please add a hyperlink for the paper? http://adsabs.harvard.edu/abs/1987PASP...99..191S 
Maybe for the whole ``Stetson in...paper`` part?